quickannotator/__main__.py

```diff
@@ -1,16 +1,17 @@
+import os
+
+os.environ['TESTING'] = 'false'  # need to set this before importing the db module
```

Comment:
hmm seems strange - this should be "False" by default if it isn't defined in the o/s? i would have expected only to require setting Testing when True - which would be done by the pytest directly?

Reply:
also seems strange that "false" is lower case?

Reply:
Looks like pytest sets the `PYTEST_CURRENT_TEST` environment variable automatically:
```python
import psutil

for pid in psutil.pids():
    environ = psutil.Process(pid).environ()
    if "PYTEST_CURRENT_TEST" in environ:
        print(f'pytest process {pid} running: {environ["PYTEST_CURRENT_TEST"]}')
```

It's probably better practice for me to use this variable directly than to manage our own variable. It also allows us to check for existence rather than casting the string to a boolean. Thoughts?

Reply:
yessssssssss, love it

Reply:
https://stackoverflow.com/questions/25188119/test-if-code-is-executed-from-within-a-py-test-session

Looks like `PYTEST_CURRENT_TEST` does not help us in our use case, because it is only set when a test is being run, not when fixtures are being set up.

I found this after confirming this behavior for myself.

This suggested solution worked for me:
```python
import sys

if "pytest" in sys.modules:
    ...
```

However we could also just set our own `TESTING` variable and check for existence. What do you prefer?